phpBB/phpbb/textreparser/plugins/poll_option.php

```diff
@@ -13,51 +13,31 @@
 
 namespace phpbb\textreparser\plugins;
 
-class poll_option extends \phpbb\textreparser\base
+class poll_option extends \phpbb\textreparser\row_based_plugin
 {
-	/**
-	* @var \phpbb\db\driver\driver_interface
-	*/
-	protected $db;
-
-	/**
-	* Constructor
-	*
-	* @param \phpbb\db\driver\driver_interface $db Database connection
-	*/
-	public function __construct(\phpbb\db\driver\driver_interface $db)
-	{
-		$this->db = $db;
-	}
-
 	/**
 	* {@inheritdoc}
 	*/
-	public function get_max_id()
+	public function get_columns()
 	{
-		$sql = 'SELECT MAX(topic_id) AS max_id FROM ' . POLL_OPTIONS_TABLE;
-		$result = $this->db->sql_query($sql);
-		$max_id = (int) $this->db->sql_fetchfield('max_id');
-		$this->db->sql_freeresult($result);
-
-		return $max_id;
+		return array(
```

Comment:
I think it is preferable to use the short array syntax here.

Reply:
I agree, and I never use the old syntax when I write code but that part was copy/pasted and I didn't notice it before I pushed it. And because every push triggers like 8 hours of Travis time, I don't want to trigger a new build for a small change. I'll modify it locally and it'll get committed if I push an update.